### ISSUE TITLE
the  hyperparameter setting for Bert model

### ISSUE THREAD

**Issue by Code-health** [zhangzheng1997/SSEGCN-ABSA#5]
could you please note the hyperparemeter setting for  the SSEGCN_BERT model? I  have ran the Bert-Based model  for many times and it  stills get unreasonable score

**Comment by Code-health** [comment]
and what's the meaning of "short" in the datafile?

**Comment by lancorrect** [comment]
> 

I think it's the shortest distance between two words in syntactic tree

**Comment by Victoriaheiheihei** [comment]
> could you please note the hyperparemeter setting for the SSEGCN_BERT model? I have ran the Bert-Based model for many times and it stills get unreasonable score

hello,
I want to follow the SSEGCN_BERT model as well. Would you tell me do you use the same run.sh to run the SSEGCN_BERT model?

**Comment by Code-health** [comment]
> > could you please note the hyperparemeter setting for the SSEGCN_BERT model? I have ran the Bert-Based model for many times and it stills get unreasonable score
> 
> hello, I want to follow the SSEGCN_BERT model as well. Would you tell me do you use the same run.sh to run the SSEGCN_BERT model?

yes, i follow the .sh file to run the model and the result is lower than what had been reported. 
I have tried  other seed but it still can't reach reported result

**Comment by Katherine536** [comment]
> could you please note the hyperparemeter setting for the SSEGCN_BERT model? I have ran the Bert-Based model for many times and it stills get unreasonable score

hello, have you successfully run the 'ssegcn_bert' model? have you seen this error
```
\SSEGCN-ABSA\models\ssegcn_bert.py", line 26, in forward
    mean = x.mean(-1, keepdim=True)
AttributeError: 'str' object has no attribute 'mean'
```
How did it work out? Thank you very much if you can answer me.
